README.md

```diff
@@ -46,11 +46,15 @@ ________________________________________________________________________________
 
 ### Ready to deploy pre-trained SOTA models
 
-YOLO-NAS architecture is out! The new YOLO-NAS delivers state-of-the-art performance with the unparalleled accuracy-speed performance, outperforming other models such as YOLOv5, YOLOv6, YOLOv7 and YOLOv8.
-Check it out here: [YOLO-NAS](YOLONAS.md).
+YOLO-NAS and YOLO-NAS-POSE architectures are out! 
+The new YOLO-NAS delivers state-of-the-art performance with the unparalleled accuracy-speed performance, outperforming other models such as YOLOv5, YOLOv6, YOLOv7 and YOLOv8.
+A YOLO-NAS-POSE model for pose estimation is also available, delivering state-of-the-art accuracy/performance tradeoff.
+
+Check these out here: [YOLO-NAS](YOLONAS.md) & [YOLO-NAS-POSE](YOLONAS-POSE.md).
 
 <div align="center">
-<img src="https://github.com/Deci-AI/super-gradients/raw/master/documentation/source/images/yolo_nas_frontier.png" width="800px">
+<img src="https://github.com/Deci-AI/super-gradients/raw/master/documentation/source/images/yolo_nas_frontier.png" height="600px">
+<img src="https://github.com/Deci-AI/super-gradients/raw/master/documentation/source/images/yolo_nas_pose_frontier_t4.png" height="600px">
```

Comment:
The width is different for both images and it looks a bit off. Maybe set `width=` instead of `height=`?

Reply:
(see here)
<img width="1120" alt="image" src="https://github.com/Deci-AI/super-gradients/assets/35190946/6b06bdb3-c3b9-4ed9-8107-c5422918105f">